R/tests/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
```

Comment:
Should we be providing usage information with the `if` function? I thought that this method was more standard?

```
#!/bin/sh
#/ Usage: test.sh <pkg_ident>
#/
#/ Example: test.sh core/tlog/6/20181108151533
#/

set -euo pipefail

if [[ -z "${1:-}" ]]; then
  grep '^#/' < "${0}" | cut -c4-
	exit 1
fi
```

Reply:
I'm sorry, I'm the one who's wrong here. It looks like the standards doc says your method is correct!

I think there's an argument for talking about improving this method, but that won't block this PR. <3